guikit/core.py

```diff
@@ -95,11 +95,28 @@ def populate_window(self):
         self._make_menubar()
         self.SetInitialSize(wx.Size(self.size))
 
+    def on_quit(self, evt):
+        """Event to close the main window form the menu."""
+        self.Close()
+
+    def populate_built_in_menu(self):
+        """Populate the menu with some built-in capabilities."""
+        return [
+            MenuTool(
+                menu="File",
+                id=wx.ID_EXIT,
+                text="Exit",
+                description="Terminate application",
+                callback=self.on_quit,
+            ),
+        ]
+
     def _make_menubar(self) -> None:
         """Create the menu bar from the entries provided by the widgets."""
         # Collecting the menu entries
         entries = itertools.chain.from_iterable(
-            [view().menu_entries() for view in KNOWN_PLUGINS]
+            [self.populate_built_in_menu()]
+            + [view().menu_entries() for view in KNOWN_PLUGINS]
```

Comment:
Does this mean that the "exit" button will always be at the top of the "File" menu? Isn't it usually at the bottom?